From a60e73484000ba107b3f7d50c622244cd9ac4c55 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Wed, 7 Feb 2024 16:32:52 +0900
Subject: [PATCH] Change optimize O2 on build system of Debian

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 tasks/toolchains/gcc.rake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tasks/toolchains/gcc.rake b/tasks/toolchains/gcc.rake
index 675c26880..81d0302d5 100644
--- a/tasks/toolchains/gcc.rake
+++ b/tasks/toolchains/gcc.rake
@@ -22,7 +22,7 @@ MRuby::Toolchain.new(:gcc) do |conf, params|
     compiler.cxx_invalid_flags = c_mandatory_flags + cxx_invalid_flags
 
     def compiler.setup_debug(conf)
-      self.flags << %w(-g3 -O0)
+      self.flags << %w(-g3 -O2)
     end
   end
 
-- 
2.43.0

